### ISSUE TITLE
SwitchRexpression Pattern Matching is not breaking

### ISSUE THREAD

**Issue by belav** [belav/csharpier#214]
The 3rd part of this is way past the printWidth but doesn't break.
```c#
var member = expression switch
{
    MemberExpression { Expression: Expression target, Member: MemberInfo propertyOrField } => new Member(
        expression,
        propertyOrField,
        target
    ),
    MethodCallExpression { Method: var instanceMethod, Object: Expression target } => new Member(
        expression,
        instanceMethod,
        target
    ),
    MethodCallExpression { Method: var extensionMethod, Arguments: { Count: > 0 } arguments } when extensionMethod.Has<ExtensionAttribute>() => new Member(
        expression,
        extensionMethod,
        arguments[0]
    ),
    _ => default
};
```

**Comment by belav** [comment]
I believe the problem is more with the IsPattern, see
```c#
            if (
                !(node
                    is PrefixUnaryExpressionSyntax { Operand: ParenthesizedExpressionSyntax { Expression: IsPatternExpressionSyntax { Pattern: DeclarationPatternSyntax, } isPattern, }, } notExpression)
            ) {
                return;
            }
```

**Comment by belav** [comment]
This will hopefully be resolved in #154

**Comment by belav** [comment]
The original issue was resolved with #382
